addon/helpers/interpolate.js

```diff
@@ -0,0 +1,10 @@
+import { helper } from '@ember/component/helper';
+import { get } from '@ember/object';
+
+export function interpolate([str], hash) {
+  const regularExpression = /\$\{\s*(.*?)\s*\}/g;
```

Comment:
if you're naming it `regularExpression` no need to assign it, right? 😛

Reply:
True 😛